File: malevich-web/src/app/_transfer/accountStateDto.ts
import {ArtworkStockDto} from "./artworkStockDto";
import {ParticipantDto} from "./participantDto";

export class AccountStateDto {
  id: number;
  participant: ParticipantDto;
  artworkStock: ArtworkStockDto;
  amount: number;
  quantity: number;
}

File: malevich-server/src/main/java/io/malevich/server/transfer/AccountStateDto.java
package io.malevich.server.transfer;

import lombok.Getter;
import lombok.Setter;


@Getter
@Setter
public class AccountStateDto {


    private Long id;

    private ParticipantDto participant;

    private ArtworkStockDto artworkStock;

    private Long quantity;

    private Double amount;

}
